docs/administration.platforms.md

```diff
@@ -1,37 +1,70 @@
 ---
 id: administration.platforms
-title: Platform Operation
+title: meshPlatforms
 ---
 
-Operators can perform various maintainance operations for their platforms. For maintenance of Landing Zones, please see
-[Landing Zone Administration](meshcloud.landing-zones.md).
+A key task for Platform Operators is integrating your Cloud Platforms as meshPlatforms into meshStack. This enables
+tenant creation, replication and metering of your Cloud Platforms. The maintenance includes
+general information about the Cloud Platform as well as structuring and configuring them.
 
-## Platform Notifications
+A central part of it also includes the application of security and governance which is covered in the
+[Landing Zone Administration](administration.landing-zones.md).
 
-Administrators can inform users about platform specific events (updates, new features, service disruptions, etc.) by creating platform
-notifications:
+Getting an insight into all tenants that are being used in a certain Cloud Platform can be found in the
+[Tenants](administration.tenants.md) section.
 
-- **Platforms**: Select the platforms relevant to your notification so they can be shown on the affected platform's dashboards. Users
-  can also [subscribe](meshcloud.profile.md#profile) to notifications by platform and [subscribed users](./meshcloud.profile.md#platform-notification-subscriptions) will receive new notifications via email.
-- **Message**: The message that will be shown to all users. You can enter valid HTML in this field (except for the `<p>` tag). This is useful for e.g. supplying links to support pages.
-- **Severity**: The type of message (info, warning, critical).
-  - Info: for new features, upcoming services updates and general information that is not expected to disrupt platform operation
-  - Warning: best used to inform about partial platform outages or problems that may occur for some users
-  - Critical: appropriate in case of complete platform failures or impending platform downtimes
-- **Show From**: The earliest time the message will be shown. You can use this to create messages that will only be shown on at a later date.
-- **Show Until**: After this time the message will no longer be displayed.
+> All platform related maintenance like Landing Zones, Tenants, etc will be moved to the [Platform Control Plane](#platform-control-plane)
+> in future.
 
-At the bottom of the screen, a preview is shown. This can give you an idea of how the message is presented to your end-users.
+## Platform Creation
+
+You can integrate new meshPlatforms in self-service via a wizard in meshStack.
+
+![Platform Creation](assets/platform_maintenance/platform-creation.png)
+
+The following platform types are available for selection.
+
+![Platform Types](assets/platform_maintenance/platform-types.png)
+
+You have to assign a meshLocation to your meshPlatform and can provide additional provider information to the meshLocation if you
```

Comment:
I would recommend to add some best practices for the meshLocation. I could imagine that users do not really know what to do with them initially. Something like:

The meshLocation is useful for grouping together multiple platforms. This is especially helpful for private cloud platforms that have an actual physical location. For public cloud platform, we recommend to name them according to the provider as public cloud platforms are not really tied to a physical location. E.g. name the meshLocation for Azure simply "Azure".